src/scripts/app.js

```diff
@@ -15,13 +15,13 @@ import LoginView      from './views/form';
 import RegisterView   from './views/form';
 import GuestLoginView from './views/form';
 import UserAccessView from './views/form';
-
 import UserAction     from './actions/user';
 
 import qs          from 'query-string';
 import FormView    from './views/form/';
 import ProfileView from './views/profile';
 
+
```

Comment:
Also, run `gulp lint` before making a pull request. This will help you get rid of most of the funky formatting issues.